src/main/java/educonnect/model/student/Link.java

```diff
@@ -9,8 +9,12 @@
 public class Link {
 
     public static final String MESSAGE_CONSTRAINTS = "Please provide a valid website.";
+
+    //    This validation regex is obtained from Mustofa Rizwan at
+    //    https://stackoverflow.com/questions/42618872/regex-for-website-or-url-validation
```

Comment:
For reused work could follow this guideline
https://nus-cs2103-ay2324s2.github.io/website/admin/appendixB-policies.html#giving-credit-for-reused-work